src/videojs.markers.js

```diff
@@ -1,9 +1,25 @@
+/*! videojs-markers - v0.4.0 - 2015-03-13
+* Copyright (c) 2015 ; Licensed  */
 /*! videojs-markers !*/
 'use strict'; 
 
 (function($, videojs, undefined) {
    //default setting
-   var defaultSetting = {
+   var defaultSetting = { 
```

Comment:
Can you explain the purpose of adding these? They don't seem to me to add new capabilities from the existing API.